rnote-ui/src/canvas/input.rs

```diff
@@ -210,6 +221,12 @@ pub(crate) fn handle_pointer_controller_event(
         }
     }
 
+    if matches!(state, PenState::Down) {
```

Comment:
do we need check for the pen state here? Unless I am missing something the input would be rejected correctly regardless and this behaviour might be confusing if `pen_input_source()` is reused elsewhere in the future.

Reply:
Thanks for pointing that out, you're right - this is unnecessary.